test/utils.h

```diff
@@ -395,7 +395,9 @@ inline bool maybeClearAllocator(int64_t max_bytes = ((int64_t)1 << 32)) {
 #if TORCH_VERSION_GREATER(2, 0, 1)
     // GetDevice was introduced in https://github.com/pytorch/pytorch/pull/94864
     // in order to properly handle new CUDA 112 behavior
-    c10::cuda::GetDevice(&device);
+    // c10::cuda uses DeviceIndex instead of int
+    // https://github.com/pytorch/pytorch/pull/119142
+    c10::cuda::GetDevice(reinterpret_cast<c10::DeviceIndex*>(&device));
```

Comment:
Hmm, why this is needed? It seems we are not using `device`. 

@jacobhinkle I remember this part was originally written by you. Looks like this is a related comment.

https://github.com/NVIDIA/Fuser/pull/390#discussion_r1202728217

I wonder `device` should be used at line 405 instead of `0`?

Reply:
Looks like getDeviceStats still uses `int`.
https://github.com/pytorch/pytorch/blob/3372aa51b4797b69db97ba124b880e891ad5a85e/c10/cuda/CUDACachingAllocator.h#L233

Reply:
You're right it should be used instead of 0. That was my mistake.

Reply:
I'll add the change to this PR.